lace/test_obj.py

```diff
@@ -188,6 +188,49 @@ def test_reading_obj_with_mtl_from_s3_url(self, mock_s3_open):
         ])
         self.assertEqual(m.materials_filepath, self.obj_with_texure_mtl)
         self.assertEqual(m.texture_filepath, self.obj_with_texure_tex)
+        self.assertIsNotNone(m.texture_image)
+
+    def test_changing_texture_filepath(self):
+        m = obj.load(self.obj_with_texure)
+        self.assertEqual(m.texture_filepath, self.obj_with_texure_tex)
+        self.assertIsNotNone(m.texture_image)
+        m.texture_filepath = None
+        self.assertIsNone(m.texture_image)
+
+    def test_texture_in_Kd_vs_Ka(self):
+        skip_if_unavailable('s3')
+        obj_Ka_path = os.path.join(self.scratch_dir, 'with_Ka_in_mtl.obj')
+        obj_Kd_path = os.path.join(self.scratch_dir, 'with_Kd_in_mtl.obj')
```

Comment:
These paths in a real OBJ file are actually relative to the obj file, not absolute paths.